### DISCUSSION TITLE
Security Policy Help

### DISCUSSION THREAD

**Discussion by NinjaJake86** [tauri-apps/tauri#3526] [Q&A] [answered]
Hi all,

Got an issue with the CSP in my tauri app, no matter what i set it to i get the error "URL not allowed on the configured scope", i've even tried setting it to null, anyone got any ideas? I am using the http api from tauri

Thanks!

**Comment by FabianLars** [comment, marked as answer]
You need to configure the scope in tauri.conf.json under [`tauri.allowlist.http.scope`](https://tauri.studio/docs/api/config#tauri.allowlist.http.scope). For a small example see here: https://tauri.studio/docs/development/security#http

**Reply by NinjaJake86** [reply]
Honestly, i should have asked sooner, thats sorted it, thank you so much!
